float64 timestamp
float32[] sensel_values
int32[] sensel_shape

float32[] commands
string commands_source

int32  counter
string id_episode

string id_robot
string id_actuators
string id_sensors
string id_environment
string commands_spec

# one of "sim", "real", "log"
string type 